include/dgl/runtime/ndarray.h

```diff
@@ -332,30 +324,41 @@ class NDArray {
   DGL_DLL static void RecordStream(DGLArray* tensor, DGLStreamHandle stream);
 
   // internal namespace
-  struct Internal;
+  struct Internal {
```

Comment:
It's still an internal struct. This modification is to put the method declarations in the header file because some methods, for example, `Internal::MoveAsDGLArray` is used in DLConveter.cc. Previously, the whole implementation of struct Internal is in ndarray.cc and cannot be accessed outside of it.